keps/sig-api-machinery/20190711-remove-selflink.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Deprecate and remove SelfLink
```

Comment:
The KEP requires extending it visibly, but I would like to first agree that this is something we even want to proceed with.